### PR TITLE
Adding Google Tag Manager

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This PR is to add Google Tag Manager to Runner. This will replace the current Google Analytics implementation 

This is linked to these PRs
https://github.com/ONSdigital/eq-terraform/pull/162
https://github.com/ONSdigital/eq-deploy/pull/69

### How to review 
Review code changes and check they make sense. Then alongside the secrets PR for eq-terraform make sure that tag manager is added when using runner by checking that tags are being fired in gtm and ua.

### CODE REVIEW

**Comment by rmccar** [comment]
[Turn on tag manager/GA for Runner (1)](https://trello.com/c/Bc2H7IbD/1021-turn-on-tag-manager-ga-for-runner-1)

File: app/templates/layouts/_base.html [hunk 1]
```diff
@@ -46,27 +46,43 @@
...
+      })(window, document, "script", "dataLayer", "{{analytics_gtm_env_id}}");
     </script>
     {% endif %}
-    <!-- End Google Analytics -->
+    <!-- End Google Tag Manager -->
   </head>
 
   <body>
 
+    <!-- Google Tag Manager (noscript) -->
```
Comment (tgandrews) [inline app/templates/layouts/_base.html:62]: This includes a `script` tag in the noscript part? huh?

Comment (rmccar) [reply app/templates/layouts/_base.html:62]: This is taken from the snippet from GTM that I had to change to allow me to pass in the environment variables and to check the environment variables exist before rendering the tags. Ill remove the "(noscript)" from the comment

**Review by tgandrews — APPROVED** [review]

**Review by heofs — APPROVED** [review]
Looks good!

**Pull request merged**